schemas/dab.draft.schema.json

```diff
@@ -175,6 +175,23 @@
             "enabled": {
               "type": "boolean",
               "description": "Allow enabling/disabling GraphQL requests for all entities."
+            },
+            "nested-mutations": {
+              "type": "object",
+              "description": "Feature flags for nested mutation operations",
+              "properties": {
+                "create":{
```

Comment:
in the pr description, i see `insert`, but here it's `create` ?

Reply:
PR description is not update. Will update

Reply:
Updated